playbooks/byo/openshift-cluster/initialize_groups.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Create initial host groups for localhost
+  hosts: localhost
+  connection: local
+  become: no
+  gather_facts: no
+  tags:
+  - always
+  tasks:
+  - include_vars: cluster_hosts.yml
+  - name: Evaluate group l_oo_all_hosts
+    add_host:
+      name: "{{ item }}"
+      groups: l_oo_all_hosts
```

Comment:
@kwoodson From what I've been able to dig out, the `l_oo_all_hosts` group was supposed to be something that was only referenced in the original playbook for creating the all `g_` group names, which are then processed into the `oo_` group names.  The group name initialization process is something I have a card for to come back to.  I've found where this `l_oo_all_hosts` group name has been used in other places, outside of the original playbook (probably by mistake), which should be corrected.